src/ExportObject.cs

```diff
@@ -148,6 +154,92 @@ public virtual void HandleMethodCall (MessageContainer method_call)
 			}
 		}
 
+		private void HandlePropertyCall(MessageContainer method_call)
+		{
+			Message msg = method_call.Message;
+			MessageReader msgReader = new MessageReader(msg);
+			MessageWriter retWriter = new MessageWriter();
+
+			object[] args = MessageHelper.GetDynamicValues(msg);
+
+			string face = (string)args[0];
+
+			if ("GetAll" == method_call.Member) {
+				conn.MaybeSendUnknownMethodError(method_call);
+				return;
+			}
+
+			string name = (string)args[1];
+
+			PropertyInfo pi = Object.GetType().GetProperty(name);
+			MethodInfo mi;
+			MethodCaller pc;
+			Signature outSig, inSig = method_call.Signature;
+
+			switch (method_call.Member) {
+				case "Set":
+					mi = pi.GetSetMethod ();
+					pc = TypeImplementer.GenSetCall(pi);
+					outSig = Signature.Empty;
+					break;
+				case "Get":
+					mi = pi.GetGetMethod ();
+					pc = TypeImplementer.GenGetCall(pi);
+					outSig = Signature.GetSig(mi.ReturnType);
+					break;
+				default:
+					throw new ArgumentException(string.Format ("No such method {0}.{1}", method_call.Interface, method_call.Member));
+			}
+
+			if (null == pc) {
+				conn.MaybeSendUnknownMethodError (method_call);
+				return;
+			}
+
+			Exception raisedException = null;
+			try {
+				pc(Object, msgReader, msg, retWriter, null);
+			} catch (Exception e) {
+				raisedException = e;
+			}
+
```

Comment:
In my master branch I had refactored this copy/paste into a function called `IssueReply` but I've avoided that change to try and keep the patch small.